tsrc/executor.py

```diff
@@ -0,0 +1,45 @@
+""" Helpers to run thing on multiple repos and collect errors """
```

Comment:
*things.  Nope, it does not only run git commands.

`tsrc sync` uses `shutil` to copy files, and `tsrc foreach` just uses subprocess directly